include/libtorrent/netlink.hpp

```diff
@@ -63,7 +63,7 @@ namespace libtorrent
 			std::memset(&m_sockaddr, 0, sizeof(sockaddr_nl));
 			m_sockaddr.nl_family = AF_NETLINK;
 			m_sockaddr.nl_groups = group;
-			m_sockaddr.nl_pid = pid;
+			m_sockaddr.nl_pid = std::uint32_t(pid);
```

Comment:
I believe the use of ``pid_t`` is a typo in the man page. If you look at the actual header, they're both u32. http://lxr.free-electrons.com/source/include/uapi/linux/netlink.h#L35

Which means that the use of ``pid_t`` here is probably incorrect as well.